zendesk_tickets/client.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+import requests
+from django.conf import settings
+
+
+TICKETS_URL = settings.ZENDESK_BASE_URL + '/api/v2/tickets.json'
```

Comment:
(Relatedly: I really want our API to fully support [HATEOAS](https://en.wikipedia.org/wiki/HATEOAS), long term.